x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/aggregation/blockhash/BlockHash.java

```diff
@@ -111,6 +111,13 @@ public static BlockHash build(List<GroupSpec> groups, BlockFactory blockFactory,
         return new PackedValuesBlockHash(groups, blockFactory, emitBatchSize);
     }
 
+    /**
+     * Temporary method to build a {@link PackedValuesBlockHash}.
+     */
+    public static BlockHash buildPackedValuesBlockHash(List<GroupSpec> groups, BlockFactory blockFactory, int emitBatchSize) {
+        return new PackedValuesBlockHash(groups, blockFactory, emitBatchSize);
+    }
```

Comment:
I need to force using a `PackedValuesBlockHash` for a bit, until I work out how to handle the different ordinals. I didn't want to make the class public, so I plumbed it through here.